mopidy_mpd/protocol/connection.py

```diff
@@ -51,3 +52,54 @@ def ping(context):
         Does nothing but return ``OK``.
     """
     pass
+
+
+@protocol.commands.add("tagtypes")
+def tagtypes(context, *args):
+    """
+    *mpd.readthedocs.io, connection settings section:*
+
+        ``tagtypes``
+
+        Shows a list of available song metadata.
+
+        ``tagtypes disable {NAME...}``
+
+        Remove one or more tags from the list of tag types the client is interested in.
+
+        ``tagtypes enable {NAME...}``
+
+        Re-enable one or more tags from the list of tag types for this client.
+
+        ``tagtypes clear``
+
+        Clear the list of tag types this client is interested in.
+
+        ``tagtypes all``
+
+        Announce that this client is interested in all tag types.
+    """
+    if not args:
+        pass
+    elif args[0] == "all":
+        context.session.tagtypes = tagtype_list.TAGTYPE_LIST[:]
+    elif args[0] == "clear":
+        context.session.tagtypes.clear()
+    elif args[0] == "disable":
+        context.session.tagtypes = [
```

Comment:
I imagine with enable/disable mpd will error if you don't specify at least one tag. Does this do nothing and return OK instead?

Reply:
After updates enable/disable without tags will do nothing and return OK.